src/features/surveys/components/surveyForm/ErrorMessage.tsx

```diff
@@ -0,0 +1,14 @@
+import Box from '@mui/material/Box';
+import { FC } from 'react';
+import messageIds from 'features/surveys/l10n/messageIds';
+import { Msg } from 'core/i18n';
+
+const ErrorMessage: FC = () => {
+  return (
+    <Box bgcolor="#ee878a" mx={4} my={4} p={2} sx={{ borderRadius: '5px' }}>
```

Comment:
Ideally the color should be retrieved from `useTheme()`.

Reply:
Fixed in 8fcde2b572a0ac78e8fbe93c5f0618bd0643df0f!